core/model.py

```diff
@@ -42,7 +42,13 @@ def __init__(self, path=DEFAULT_MODEL_PATH):
         logger.info('Loaded models')
 
     def read_image(self, image_data):
-        return Image.open(io.BytesIO(image_data)).convert("RGB")
+        try:
+            image = Image.open(io.BytesIO(image_data))
+            if image.mode != 'RGB':
+                image = image.convert('RGB')
+            return image
+        except IOError:
+            abort(400, 'Invalid file type/extension. Please provide a valid image (supported formats: JPEG, PNG, TIFF).')
```

Comment:
The README says the supported formats are `Image (RGB/HWC)`. I think we should be consistent